src/main/java/org/alliancegenome/curation_api/services/base/BaseOntologyTermService.java

```diff
@@ -300,4 +300,15 @@ private void handleCrossReferences(OntologyTerm dbTerm, OntologyTerm incomingTer
 		}
 	}
 
+	public <T extends OntologyTerm> T findSubsetTerm(T childTerm, String subsetName) {
+		for (OntologyTerm term : childTerm.getIsaAncestors()) {
```

Comment:
Is this always going to be an ancestor term that we want returned?  Or could it be the case that the term itself belongs to the subset and can just be returned?

Reply:
I think it will always be an ancestor. The [loader code](https://github.dev/alliance-genome/agr_loader/blob/2c3dcf03c12df45cc95ddc2f7abb41469836e40f/src/etl/expression_ribbon_etl.py#L43) that this is replacing  traverses  the PART_OF|/S_A relationship to get the ancestors.